terraform/staging/main.tf

```diff
@@ -87,7 +87,7 @@ module "application" {
   sidekiq_task_desired_count = 2
   sidekiq_task_memory        = 8192
 
-  ecr_repository_url = "815624722760.dkr.ecr.eu-west-2.amazonaws.com/core:staging-20240416114809"

```

Comment:
don't think I needed to commit this change in the previous branch to begin with, just reverting that here